src/functions-and-arrays.js

```diff
@@ -52,15 +100,39 @@ const wordsUnique = [
   'bring'
 ];
 
-function uniquifyArray() {}
-
-
+function uniquifyArray(array5) {
+  if (array5.length === 0) {
+    return null;
+  }
+  const newUniqueArr = [];
+  for (let i = 0; i < array5.length; i++) {
+    if (array5.indexOf(array5[i]) === array5.lastIndexOf(array5[i])) {
+      newUniqueArr.push(array5[i]);
+    }
+  }
+  return newUniqueArr;
+}
+console.log(uniquifyArray(wordsUnique))
 
 // Iteration #6: Find elements
+
 const wordsFind = ['machine', 'subset', 'trouble', 'starting', 'matter', 'eating', 'truth', 'disobedience'];
 
-function doesWordExist() {}
+function doesWordExist(keyWord, array6) {
+  if (array6.length === 0) {
+    return null;
+  }
+  for (let i = 0; i < array6.length; i++) {
+    if (array6[i] === keyWord) {
+      console.log('True')
+      break;
+    }
+    else
+    console.log('False')
```

Comment:
There is no need for an else here. After the for loop ends, if we did not find the keyWord, we should return false.

```
function doesWordExist(array6, keyWord) {
  if (array6.length === 0) {
    return null;
  }
  for (let i = 0; i < array6.length; i++) {
    if (array6[i] === keyWord) {
      return true;
    }
  }
  return false;
}
```